articles/service-fabric/service-fabric-containers-volume-logging-drivers.md

```diff
@@ -164,11 +165,13 @@ In the **Volume** element in the snippet above, the Azure Files volume plugin re
 
 As shown in the **DriverOption** elements in the snippet above, the Azure Files volume plugin supports the following driver options:
 
+Required driver option:
 - **shareName** - Name of the Azure Files file share that provides the volume for the container
 - **storageAccountName** - Name of the Azure storage account that contains the Azure Files file share
 - **storageAccountKey** - Access key for the Azure storage account that contains the Azure Files file share
 
-All of the above driver options are required.
+Optional driver option:

```

Comment:
Suggest using plural "options" because even though there is only one at the moment, we might add more in the future.